db/migrate/20220113213921_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, :post
```

Comment:
- [ ] This line also should be adjusted as following:
```suggestion
      t.references :author, foreign_key: { to_table: 'users' }
      t.references :post, foreign_key: true
```

Reply:
Thanks, I have updated them.